### DISCUSSION TITLE
commits not showing up on github contributions graph

### DISCUSSION THREAD

**Discussion by The-CoderBoy** [community/community#64947] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello 
A few weeks ago, I changed my GitHub user name. After that, my commits stop showing up on the contribution graph. I can see the repository is updated, but there is nothing on the contribution graph. i have changed my user name on my local machine using `git config --global user.name "USER_NAME" `  and try again but still nothing on contribution graph

**Comment by paras-017** [comment]
Okay, after doing google . I found the solution  run this command : git config user.email "your-email@example.com"  and then do git push --force

**Reply by The-CoderBoy** [reply]
thanks its working now
